Packages/MIES/MIES_AsynchronousData.ipf

```diff
@@ -21,12 +21,13 @@ Function ASD_ReadChannel(device, channel)
 	variable channel
 
 	string ctrl
-	variable gain, deviceChannelOffset, rawChannelValue
+	variable gain, deviceChannelOffset, rawChannelValue, hardwareType
 
 	NVAR deviceID = $GetDAQDeviceID(device)
 	deviceChannelOffset = HW_ITC_CalculateDevChannelOff(device)
 
-	rawChannelValue = HW_ReadADC(HARDWARE_ITC_DAC, deviceID, channel + deviceChannelOffset)
+	hardwareType = GetHardwareType(device)
```

Comment:
@t-b With this change async channels will work on NI hardware now?

Reply:
Yes. The test `AsyncAcquisitionLBN` checks that. And also future similiar problems are mitigated as:
- HW_ITC_ReadADC: Returns NaN on errors calling the ITC XOP2 routines. I've added a workaround so that we don't need an updated ITCXOP2, as I thought these changes would be rather involved. Turns out it was not so difficult [1].
- ASD_CheckAsynAlarmState: Treats an ADC value of NaN as failure.

[1]: So I fixed it already in https://github.com/AllenInstitute/ITCXOP2/pull/20 and https://github.com/AllenInstitute/ITCXOP2/pull/21. The ITCXOP now also follows the "modern" approach of using cmake instead of Visual Studio project files. I've been using cmake for every XOP since directly after the ITCXOP2 ;)